test/index.test.ts

```diff
@@ -63,9 +63,9 @@ describe('system test', () => {
     scope.done();
   });
 
-  it('should use GCE_METADATA_IP if available', async () => {
```

Comment:
Can we preserve a single test that ensures `GCE_METADATA_IP` continues to work?  We still want to cover it, since some folks are relying on it